src/dms/core/EdFi.DataManagementService.Core.External/Model/EducationOrganizationId.cs

```diff
@@ -6,4 +6,4 @@
 namespace EdFi.DataManagementService.Core.External.Model;
 
 // An int type branded as a EducationOrganizationId, which is a unit of security authorization.
-public record struct EducationOrganizationId(long Value);
+public record struct EducationOrganizationId(string Value);
```

Comment:
EducationOrganizationId is string now to handle bigger edorgid values in securityelements={"Contact": [], "Student": [], "Namespace": [], "EducationOrganization": [{"Id": 301, "ResourceName": "LocalEducationAgency"}, {"Id": 30101999999, "ResourceName": "SchoolId"}]}